example/src/screens/Preview.tsx

```diff
@@ -122,65 +123,61 @@ export const Preview = ({ navigation, route }: Props) => {
 
   return (
     <BackgroundAnimation>
-      <View style={styles.content}>
-        <View style={styles.header}>
-          <Typo variant="h4">Videoconferencing for everyone</Typo>
-        </View>
-        <View style={styles.titleLabel}>
-          <Typo variant="chat-regular" color={TextColors.description}>
-            {getTitleLabel()}
-          </Typo>
-        </View>
-
-        <View style={styles.cameraPreview}>
-          {isCameraOn ? (
-            <Membrane.VideoPreviewView
-              style={styles.membraneVideoPreview}
-              mirrorVideo
-              captureDeviceId={currentCamera?.id}
-            />
-          ) : (
-            <View style={styles.noCameraBackground}>
-              <View style={styles.noCameraContent}>
-                <Typo variant="h5" color={BrandColors.darkBlue80}>
-                  {getShortUsername(username)}
-                </Typo>
-              </View>
-            </View>
-          )}
+      <SafeAreaView style={{ flex: 1 }} edges={['bottom']}>
```

Comment:
I'm wondering if we'd like to have a `ScreenWrapper` component with `SafeAreaView` and maybe something else in the future. If we'd have to change the safe area we'd need to change in every screen again.